src/objects/LOD.d.ts

```diff
@@ -12,8 +12,10 @@ export class LOD extends Object3D {
 	levels: { distance: number; object: Object3D }[];
 	autoUpdate: boolean;
 	readonly isLOD: true;
+	readonly _currentLevel: 0;
```

Comment:
Can you please remove this line? We usually do not respect private properties in TS files.

Reply:
Done